frontend/packages/topology/src/components/page/TopologyPage.tsx

```diff
@@ -93,16 +110,16 @@ export const TopologyPage: React.FC<TopologyPageProps> = ({
           exact: true,
         })
       ? TopologyViewType.graph
-      : isTopologyViewStateLoaded && (topologyViewState || defaultViewType);
+      : loaded && ((getTopologyViewState() as TopologyViewType) || defaultViewType);
```

Comment:
Why the cast to `TopologyViewType` here? the function should return the appropriate type.